components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -823,6 +823,10 @@ void RewardsServiceImpl::OnLedgerInitialized(
   if (!ready_->is_signaled()) {
     ready_->Signal();
   }
+
+  for (auto& observer : observers_) {
+    observer.OnRewardsInitialized(this);
+  }
```

Comment:
It looks like `is_ledger_initialized_` above is only written and never read. If that's right, can we go ahead and remove it?

Reply:
Good catch, removed.